### DISCUSSION TITLE
Maltrail Implementation

### DISCUSSION THREAD

**Discussion by 0x023** [Security-Onion-Solutions/securityonion#4407] [Ideas]
Hi,

I was wondering if it would be interesting to look into implementing maltrail in the security onion ecosystem?
https://github.com/stamparm/MalTrail

Looking forward to an open discussion :)

**Comment by weslambert** [comment]
Are you looking to use it to tag or generate alerts for certain traffic that matches certain criteria? Have you also considered using something like MISP integration?

Ex. https://docs.securityonion.net/en/2.3/misp.html (https://github.com/weslambert/securityonion-misp)
